crates/ra_ide_api/src/change.rs

```diff
@@ -246,11 +245,6 @@ impl RootDatabase {
 
         if self.last_gc_check.elapsed() > GC_COOLDOWN {
             self.last_gc_check = crate::wasm_shims::Instant::now();
```

Comment:
This seems a bit misleading now.

Reply:
Yeah, this is here just to keep the infra if we decide to re-enable GC. And it's sort-of ok even: previously, we checked for gc once in a while, and the check was "do we have a lot of trees?". Now, the check is `return false`;